packages/console/src/pages/GetStarted/hook.ts

```diff
@@ -107,6 +107,7 @@ const useGetStartedMetadata = ({ checkDemoAppExists }: Props) => {
       subtitle: 'get_started.card5_subtitle',
       icon: isLightMode ? OneClick : OneClickDark,
       buttonText: 'admin_console.general.set_up',
+      isComplete: settings?.configureSocialSignIn,
```

Comment:
configured?

Reply:
Yes, we have talked about this before. Will do it in another PR